variable/energy/energy-secondary.yaml

```
@@ -41,18 +41,64 @@ Secondary Energy|Electricity|Fossil:
    unit: EJ/yr
 
 Secondary Energy|Electricity|Gas:
-   description: Net electricity production from natural gas
+   description: Net electricity production from methane
    unit: EJ/yr
 
 Secondary Energy|Electricity|Gas|w/ CCS:
-   description: Net electricity production from natural gas with a co2 capture
+   description: Net electricity production from methane with a co2 capture
                 component
    unit: EJ/yr
 
 Secondary Energy|Electricity|Gas|w/o CCS:
+   description: Net electricity production from methane with freely vented
+                co2 emissions
+   unit: EJ/yr
+   
+Secondary Energy|Electricity|Gas|Natural Gas:
```

suggestion for clarity
```suggestion
Secondary Energy|Electricity|Gas|Fossil:
```